crates/mfem-sys/examples/ex1.rs

```diff
@@ -32,8 +32,9 @@ use std::{ffi::CStr, pin::Pin, ptr, slice};
 
 use autocxx::prelude::*;
 use clap::Parser;
-use cxx::{let_cxx_string, UniquePtr};
+use cxx::let_cxx_string;
 use mfem_sys::*;
+use autocxx::prelude::UniquePtr;
```

Comment:
That should be the same thing, no?  (The “source” at the top of [autocxx::prelude::UniquePtr](https://docs.rs/autocxx/0.27.1/autocxx/prelude/struct.UniquePtr.html) redirects to `cxx`.)

Reply:
Yep, I realized that later on.
The actual issue is that the `cxx` API changes between versions that are all `1.*`:
In 1.0.122, `UniquePtr` does _not_ have the method `as_mut_ptr()`.
In 1.0.137, it does.